rfcs/sliders-component.md

```diff
@@ -0,0 +1,122 @@
+# Sliders Component
+
+Represents control with sliding axis to choose a single value or range between min and max values.
+
+## Usage
+
+### Basic usage
+
+```javascript
+import * as React from 'react';
+import {Sliders} from 'baseui/sliders';
+
+export default () => <Sliders range={[
+  {value: 0, label: '0, min'},
+  {value: 100, label: '100, max'}
+]} />;
```

Comment:
Can we show the "selectedValues" here too? Seems like that still belongs as part of basic usage for a stateless component.

Reply:
Also, maybe we can use the numeric Point syntax to for the basic example
```js
<Slider range={[0, 100]} />
```

Reply:
Numeric is also mentioned as `Point` type can be primitive. I can rephrase.
`selectedValues` is the result. It's optional if we don't want to provide initial value for it. We will just put thumb to min and max